### DISCUSSION TITLE
Icon for Microsoft Teams

### DISCUSSION THREAD

**Discussion by Carssons** [elrumo/macOS_Big_Sur_icons_replacements#1329] [Icon request] [answered]
Hello everyone,

I love the icons in the following style (see attached). Please could someone make an icon for Microsoft Teams in a similar style with a "T" inside?

Thank you in advance!

Regards.

<img width="1151" alt="Screen Shot 2021-11-22 at 8 32 20 AM" src="https://user-images.githubusercontent.com/86297154/142789636-b426a440-72b3-4a0d-b911-3e53788d5dc2.png">

**Comment by bunkerdesign** [comment, marked as answer]
Here you go

[teams.icns.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/7603735/teams.icns.zip)

**Reply by Carssons** [reply]
Thank you so much, bro. The icon looks great. I love it.
All the best.

**Reply by bunkerdesign** [reply]
You are welcome! :)

**Reply by bunkerdesign** [reply]
Could you please mark the request as answered?

**Reply by Carssons** [reply]
I have. Thank you ^^
